### PR TITLE
add bxslider to v4.2.5 with git autoupadte config

### PR DESCRIPTION
**Pull request by LeaYeh**

### CODE REVIEW

File: ajax/libs/jquery.bxslider/package.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    "responsive",
+    "carousel",
+    "bxslider",
+    "jQuery",
+    "plugin"
+  ],
+  "private": true,
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/stevenwanderski/bxslider-4.git",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.bxslider/package.json]: Sould use git protocol

File: ajax/libs/jquery.bxslider/package.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    "basePath": "dist",
+    "files": [
+      "jquery.bxslider.min.js",
+      "jquery.bxslider.js",
+      "jquery.bxslider.css",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.bxslider/package.json]: missing `jquery.bxslider.min.css`

**Comment by PeterDaveHello** [comment]
There are `images` and `vendor`, but didn't exist in your pr.

**Comment by PeterDaveHello** [comment]
Inconsistent file structure with origin repo.

**Comment by LeaYeh** [comment]
rename "name": "bxslider-4" and parent folder.

**Comment by LeaYeh** [comment]
rename filename and parent folder's name.

**Comment by PeterDaveHello** [comment]
Even worth, there are duplicated files, and the orl problem still exist.

**Comment by PeterDaveHello** [comment]
Any updates?

**Comment by Piicksarn** [comment]
ping @cdnjs/intern @cdnjs/intern2

**Comment by pvnr0082t** [comment]
`bxslider` had already been added. ref: c4b8897fd14266e4c09ca4b5b242e5eb807e063d. I'm going to close this PR.

**Pull request closed without merging**